torchgeo/datasets/so2sat.py

```diff
@@ -24,18 +24,24 @@ class So2Sat(NonGeoDataset):
     acquired by the Sentinel-1 and Sentinel-2 remote sensing satellites, and a
     corresponding local climate zones (LCZ) label. The dataset is distributed over
     42 cities across different continents and cultural regions of the world, and comes
-    with a split into fully independent, non-overlapping training, validation,
-    and test sets.
+    with in a variety of different splits.
```

Comment:
```suggestion
    with a variety of different splits.
```
Or some other grammar change